src/config.js

```diff
@@ -2,7 +2,16 @@
 // this will compile on-the-fly the `api.js` required below
 // by `require(PROJECT_ROOT + '/api');`
 
-export const BABEL_CONFIG = {
+// Support for `babel` property in project package.json to extend base babel config
+// to add plugins and more presets, etc. React, JSX Transform, Syntax support
+const babelRequiredPkgJson = require(process.cwd() + '/package.json');
```

Comment:
Could you update this to be `require(path.join(process.cwd(), 'package.json'))`?